### PR TITLE
4862 - Fix pager to use tabs instead arrow keys

### LINKED ISSUE #4862: Pager: accessibility issues
**Describe the bug**
There are two accessibility issues found in the soho-standalone-pager component:
1. The page size button element with aria-haspopup="true" cannot be activated with the Down Arrow or Up Arrow.
2. The soho page buttons appear to use a roving tabindex rather than individually receiving keyboard focus with the tab key. There is no role or visual indication for this functionality. 
3. the tooltip for the first and previous pages disapper after clicking next from the first page

**To Reproduce**
on your local enterprise-ng
1. run npm start
2. Go to http://localhost:4200/ids-enterprise-ng-demo/datagrid-standalone-pager
3. navigate to the pager buttons via keyboard, you cannot navigate from one button to another button via tab, but you can with arrow keys.
4. also when you navigate to the select page size button via keyboard, clicking down doesn't activate the button.
5. also after clicking next, the first and previous button should enable, try hovering your mouse or navigate your keyboard to the first and previous buttons, the tooltip doesnt show up any more.

**Expected behavior**
[VPAT Recommendations]
1. regarding the page size button: Allow Down Arrow and Up Arrow functionality to open the menu. The Down Arrow should function similarly to the Enter or Spacebar. The Up Arrow should open the menu and focus on the last menu item. 
2. regarding the first, previous, next and last buttons: Remove the arrow key functionality and ensure that each of these controls can receive focus with the Tab key instead. 

**Version**
- ids-enterprise-ng: 9.2.0-dev

**Platform**
 - Device (if applicable) :NA
 - OS Version: Windows 10
 - Browser Name: Firefox
 - Browser Version: v85.0

**Additional context**
https://jira.lawson.com/browse/LMCLIENT-34407

### PR DESCRIPTION
**Pull request by deep7102**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
- Fixed accessibility issue to use tabs instead arrow keys with Pager.
- Fixed pagesize pull down to open with arrow down key
- For disapper tooltip will handle separate issue #5161 

**Related github/jira issue (required)**:
Closes #4862

**Steps necessary to review your pull request (required)**:
Pull this branch and build/run the demo app
- Navigate to: http://localhost:4000/components/pager/example-standalone.html
- Pager button should be able to navigate by tab key
- Click to activate the setting `Show Page Size Selector`
- Use keyword to navigate that pull down in pager bar
- See it should be able to open by down arrow key

Test NG
- Pull and build NG branch ([main](https://github.com/infor-design/enterprise-ng.git))
- Copy and replace "dist" folder from `enterprise` to `⁨enterprise-ng⁩/node_modules⁩/ids-enterprise`
- Navigate to http://localhost:4200/ids-enterprise-ng-demo/datagrid-standalone-pager
- Pager button should be able to navigate by tab key
- Use keyword to navigate pagesize pull down in pager bar
- See it should be able to open by down arrow key
**Included in this Pull Request**:
- [x] A note to the change log.

### CODE REVIEW

File: src/components/pager/pager.js [hunk 1]
```diff
@@ -104,7 +105,8 @@ const PAGER_DEFAULTS = {
   pageSizeMenuSettings: {
     attachToBody: false
   },
-  attributes: null
+  attributes: null,
+  tabable: true
```
Comment (EdwardCoyle) [inline src/components/pager/pager.js:109]: Maybe want to switch this to `tabbable` throughout to be consistent with other components

**Review by EdwardCoyle — COMMENTED** [review]
@deep7102 looks good, just had a comment:

File: src/components/pager/pager.js (hunk 1 above)
Comment (deep7102) [reply src/components/pager/pager.js]: Thanks! @EdwardCoyle, update now

**Review by EdwardCoyle — APPROVED** [review]
Looks great 👍

**Pull request merged**